profile.md

```diff
@@ -178,3 +179,98 @@ part of a Kubernetes API call:
   }
   ```
 
+## Service Metadata
+
+While the [specification](spec.md) does not mandate the property names used
+in the `metadata` objects, is it RECOMMENDED that the following names
+be used when possible in an attempt to provide some degree of interoperability
+and consistency.
+
+### Service Metadata Fields
+
+| Broker API Field | Type | Description |
+| --- | --- | --- |
+| name | CLI string | A short name for the service to be displayed in a catalog. |
```

Comment:
Should define what a "CLI string" is in the description like in #248 and change the type to just `string`